### PR TITLE
Fix errors

### PR DESCRIPTION
**Pull request by xepozz**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️/❌
| New feature?  | ✔️/❌
| Breaks BC?    | ✔️/❌
| Fixed issues  | comma-separated list of tickets # fixed by the PR, if any

### CODE REVIEW

**Comment by xepozz** [comment]
It locks getting debug entries by ID, but not all entries.

File: src/Controller/ComposerController.php [hunk 1]
```diff
@@ -66,9 +66,10 @@ public function inspect(ServerRequestInterface $request, Aliases $aliases): Resp
 
     public function require(ServerRequestInterface $request, Aliases $aliases): ResponseInterface
     {
-        $package = $request->getParsedBody()['package'] ?? null;
-        $version = $request->getParsedBody()['version'] ?? null;
-        $isDev = $request->getParsedBody()['isDev'] ?? false;
+        $parsedBody = \json_decode($request->getBody()->getContents(), true, 512, JSON_THROW_ON_ERROR);
```
Comment (samdark) [inline src/Controller/ComposerController.php:69]: Why didn't `getParsedBody()` work for you?

Comment (xepozz) [reply src/Controller/ComposerController.php:69]: I don't know, seems something broken in `yii-runner-http`

Comment (vjik) [reply src/Controller/ComposerController.php:69]: [ServerRequestFactory](https://github.com/yiisoft/yii-runner-http/blob/master/src/ServerRequestFactory.php) from runner used array `$_POST` for fill parsed body, but in apps and [demo](https://github.com/yiisoft/demo/blob/master/blog-api/config/common/di/psr17.php#L7) used [ServerRequestFactory](https://github.com/httpsoft/http-message/blob/master/src/ServerRequestFactory.php) that not parse body and use `null` for it.

We can't rely on parsed body here.

Comment (samdark) [inline src/Controller/ComposerController.php:69]:
```suggestion
        // Request factory may be unable to parse JSON so don't rely on getParsedBody().
        $parsedBody = \json_decode($request->getBody()->getContents(), true, 512, JSON_THROW_ON_ERROR);
```

Comment (vjik) [reply src/Controller/ComposerController.php:70]: See https://github.com/yiisoft/yii-debug-api/pull/118#pullrequestreview-1505427738

**Comment by samdark** [comment]
I think it is OK as is.

**Pull request merged**